### PR TITLE
MINOR: Used Kafka native constants in KafkaLog4jAppender

### PR DESCRIPTION
**Pull request by kamalcph**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5336/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5351/
Test PASSed (JDK 7 and Scala 2.11).

File: log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java [hunk 1]
```diff
@@ -256,7 +254,7 @@ protected void append(LoggingEvent event) {
             try {
                 response.get();
             } catch (InterruptedException | ExecutionException ex) {
-                throw new RuntimeException(ex);
+                throw new KafkaException(ex);
```
Comment (ijuma) [inline log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java]: What is the reason for this change?

Comment (kamalcph) [reply log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java]: The error was returned by the KafkaProducer,    so I've wrapped it in KafkaException.

Comment (kamalcph) [reply log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java]: Shall I change it back to `RuntimeException` ?

@ijuma 
Please review again.

Comment (ijuma) [reply log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java]: It seems unrelated to the other changes, so maybe revert it.

Comment (kamalcph) [reply log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java]: I've reverted this one now.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6054/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6039/
Test PASSed (JDK 8 and Scala 2.12).

**Review by ijuma — APPROVED** [review]
LGTM, thanks for the PR.
